templates/ldap-master.conf.j2

```diff
@@ -0,0 +1,34 @@
+# {{ ansible_managed }}
+{%- macro dovebool(val) %}{% if val %}yes{% else %}no{% endif %}{% endmacro %}
```

Comment:
Move this to a separate file and include it

Reply:
I don't see any reason to